src/spork.cpp

```diff
@@ -238,7 +238,7 @@ bool CSporkMessage::Sign(const std::string& strSignKey)
         }
 
         if (!CHashSigner::VerifyHash(hash, pubkey, vchSig, strError)) {
-            LogPrintf("CSporkMessage::CheckSignature -- VerifyHash() failed, error: %s\n", strError);
+            LogPrintf("CSporkMessage::Sign -- VerifyHash() failed, error: %s\n", strError);
```

Comment:
Not important for this case specifically, but I'd suggest to use `LogPrintf("%s -- some debugging", __func__)` in the future. I started doing this after I struggled too often when copying/moving debug prints.

Reply:
Yeah, I'd like to refactor all this one day... The issue is `__func__` won't give you a class name, so you still have to do smth like `CSomeClass::%s` to get more or less helpful log output.